src/Microsoft.ML.StandardLearners/Standard/MultiClass/Ova.cs

```diff
@@ -32,19 +32,21 @@
 [assembly: EntryPointModule(typeof(OvaPredictor))]
 namespace Microsoft.ML.Runtime.Learners
 {
+    using TScalarPredictor = IPredictorProducing<float>;
+    using TScalarTrainer = ITrainerEstimator<IPredictionTransformer<IPredictorProducing<float>>, IPredictorProducing<float>>;
```

Comment:
>IPredictorProducing<float> [](start = 68, length = 26)

can we change that to `TScalarPredictor` ?

Reply:
can't use TScalarPredictor yet: type could not be found. 

---
In reply to: [217234499](https://github.com/dotnet/machinelearning/pull/865#discussion_r217234499) [](ancestors = 217234499)